go/vendor/github.com/keybase/go-crypto/ssh/terminal/terminal.go

```diff
@@ -1,899 +0,0 @@
-// Copyright 2011 The Go Authors. All rights reserved.
```

Comment:
We need to leave this in.  See [this commit](https://github.com/keybase/go-crypto/commit/0829a5f1b577747f1935b43155b575f397c3848a)